sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -242,6 +242,22 @@ cmake -DIntel_SYCL_ROOT=$SYCL_HOME/deploy -DSYCL_IMPLEMENTATION=Intel_SYCL ...
 cmake -DIntel_SYCL_ROOT=%SYCL_HOME%\deploy -DSYCL_IMPLEMENTATION=Intel_SYCL ...
 ```
 
+### Build Doxygen documentation
+
+Building Doxygen documentation is similar to building the product itself. First,
+you need to install requirements:
+- doxygen
+- graphviz
+
+Then you'll need to add the following options to your CMake configuration
+command:
+```
+-DLLVM_ENABLE_DOXYGEN=ON
+```
+
+After CMake cache is generated, build the documentation with `doxygen-sycl`
+target. It will be put to `/path/to/build/tools/sycl/doc/html` directory.
+
 ## Run simple SYCL application
 
 A simple SYCL program consists of following parts:
```

Comment:
please include doc generation in regular build to make sure that it is checked in all pre-commit jobs.

Reply:
Personally, I think that separate CI jobs for docs is preferable, because doxygen is is OS-agnostic, we don't need to re-build docs on Windows

Reply:
I would like to avoid submitting that until we have test enabled.
Please contact @tfzhu  if you need any support here.

Reply:
> I would like to avoid submitting that until we have test enabled.
> Please contact @tfzhu if you need any support here.

Could you clarify why? I don't see why this commit should be blocked by enabling CI. Also I think it can be handled by @alexbatashev without any help, but later.